docs/fundamentals/wadm/deploying.md

```diff
@@ -2,13 +2,211 @@
 title: "Deploying Wadm"
 date: 2020-01-19T00:00:00+00:00
 icon: "ti-map" # themify icon pack : https://themify.me/themify-icons
-description: "Deploying and Running the Wadm Application in a Cluster"
+description: "Deploying and Running the Wadm Application"
 type: "docs"
 sidebar_position: 1
```

Comment:
IMO this should be page 4: after the intro, defining, and deploying apps, and before the API reference page